Tests/FuzzilliTests/MutationsTest.swift

```diff
@@ -35,12 +35,23 @@ extension BaseInstructionMutator {
 class MutationsTests: XCTestCase {
 
     func testPrepareMutationRuntimeTypes() {
-        let fuzzer = makeMockFuzzer()
+        /// The mutation fuzzer responsible for mutating programs from the corpus and evaluating the outcome.
+        let mutators = WeightedList<Mutator>([
+            (CodeGenMutator(),   1),
+            (OperationMutator(), 1),
+            (InputMutator(),     1),
+            (CombineMutator(),   1),
+            (JITStressMutator(), 1),
+        ])
+
+        let engine = MutationEngine(mutators: mutators, numConsecutiveMutations: 5)
```

Comment:
How about you let `makeMockMutationEngine` return a `MutationEngine`, then you can replace these lines with just a call to `makeMockMutationEngine()`